self-tests.yml

```diff
@@ -227,6 +227,8 @@ functions:
         OTEL_COLLECTOR_ENDPOINT: ${otel_collector_endpoint}
         OTEL_TRACE_ID: ${otel_trace_id}
         OTEL_PARENT_ID: ${otel_parent_id}
+        MONGOTOOLS_URL: ${mongotools_url}
+        MONGOTOOLS_DECOMPRESS: ${mongotools_decompress|*decompress}
```

Comment:
It seems like `mongotools_decompress` is never set. Do we need a separate decompress env var just for mongo-tools if it's always just `decmopress`?

Reply:
Some are zip other tar. Right now we are only downloading tar but if we decide to download for other variants we may need to unzip instead. I can remove it and we can always add it back in if needed?

Reply:
Makes sense. I'm alright with just leaving it as-is since it doesn't really add any complexity to have a separate env var.